.agents/skills/shadcn-vue/assets/tailwind.css

```
@@ -0,0 +1,73 @@
+@import "tailwindcss";
+
+@tailwind base;
```

Stylelint (17.10.0)

[error] Unexpected unknown at-rule "@tailwind" (scss/at-rule-no-unknown)

(scss/at-rule-no-unknown)